README.md

```diff
@@ -1,3 +1,5 @@
+[![Codacy Badge](https://app.codacy.com/project/badge/Grade/0642a6f7d92542e39f2dd68139cd5586)](https://www.codacy.com/gh/Seagate/cortx-hare/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Seagate/cortx-hare&amp;utm_campaign=Badge_Grade)
```

Comment:
Problem: Codacy Badge needs to be included in README.md file , which provides code quality status for every commit in the repository.It also contains link to the repository in Codacy.
Solution: Codacy badge link is copied from the settings tab under Codacy Hare repository and pasted in README.md file.